src/mixins/view.js

```diff
@@ -129,7 +129,7 @@ const ViewMixin = {
     this.unbindUIElements();
 
     // remove the view from the DOM
-    this.removeEl();
```

Comment:
@blikblum I just realized this was introducing new API to view that was only there before because of the DOM Mixin.  https://github.com/marionettejs/backbone.marionette/issues/3379